symfony/framework-bundle/3.3/web/index.php

```diff
@@ -23,6 +23,7 @@
     if (isset($_SERVER['HTTP_CLIENT_IP'])
         || isset($_SERVER['HTTP_X_FORWARDED_FOR'])
         || !(in_array(@$_SERVER['REMOTE_ADDR'], ['127.0.0.1', '::1']) || PHP_SAPI === 'cli-server')
+        && getenv('APP_ENV') !== 'prod'
```

Comment:
This is only ever reached when the `APP_DEBUG` environment variable is set which should not be the case in production.